packages/bootstrap/scss/popup/_theme.scss

```diff
@@ -12,6 +12,15 @@
             font-size: $font-size-sm;
         }
 
+        .k-outer-group-header {
+            border-top-color: $grouping-border;
+            font-size: .875rem;
+            color: $frozen-grouping-header-color;
+        }
+        &.k-first {
```

Comment:
`.k-first` class is at the same element as `.k-outer-group-header`, and not with the `.k-popup` element. This style is not applying and can be removed

Reply:
Removed.